src/App.tsx

```diff
@@ -1,17 +1,130 @@
+import { useState, useEffect } from "react";
 import Header from "./Components/Header";
-import InputTodo from "./Components/InputTodo";
-import TodoList from "./Components/TodoList";
+import TodoInput from "./Components/TodoInput";
+import TodoItem from "./Components/TodoItem";
+import CreatePortal from "./Components/CreatePortal";
+import { useAppDispatch, useAppSelector } from "./hooks";
+import { selectItem } from "./Features/textItem/item-slice";
+import { IItem } from "./Features/textItem/item-slice";
+/* import { textItemTEST } from "./Features/Item/textItem-slice"; */
+
+import {
+  addItem,
+  /*  editItem, */
+  deleteItem,
+  completedItem,
+} from "./Features/textItem/item-slice";
+
+/* interface ITodos {
+  id: number;
+  todo: string;
+  completed: boolean;
+} */
 
 const App: React.FC = () => {
+  const [todos, setTodos] = useState<IItem[]>([]);
```

Comment:
Do we really need local state for todos here? I guess we should use redux store as only source of todo items